src/uct/tcp/tcp_ep.c

```diff
@@ -140,11 +138,19 @@ void uct_tcp_ep_mod_events(uct_tcp_ep_t *ep, uint32_t add, uint32_t remove)
     int new_events = (ep->events | add) & ~remove;
 
     if (new_events != ep->events) {
+        int prev = ep->events;
```

Comment:
unaligned '='
all declarations at begin of func

Reply:
thx! done